frontend/src/styles/vars.scss

```diff
@@ -372,6 +372,10 @@ $_lifecycle_dormant: $_danger;
     --modal-backdrop-color: rgb(0 0 0 / 20%);
     --modal-transition-time: 200ms;
 
+    // Tooltips
+    --tooltip-background-light: var(--bg-charcoal);
+    --tooltip-background-dark: #656d81;
```

Comment:
Great foundation for tuning this in the future.
Note: considering we're using `--tooltip-bg`, these should be `--tooltip-bg-{light,dark}`